azext_iot/tests/iothub/devices/test_hub_export_import_int.py

```diff
@@ -0,0 +1,496 @@
+import random
+import pytest
+import json
+import os
+import time
+from pathlib import Path
+from azext_iot.common.shared import DeviceAuthApiType
+
+from azext_iot.tests.settings import DynamoSettings, ENV_SET_TEST_IOTHUB_REQUIRED, ENV_SET_TEST_IOTHUB_OPTIONAL
+from azext_iot.tests.iothub import IoTLiveScenarioTest
+from azext_iot.tests.generators import generate_generic_id
+from azext_iot.common.utility import generate_key, read_file_content
+from azext_iot.common.certops import create_self_signed_certificate
+from azext_iot.tests.iothub import (
+    PRIMARY_THUMBPRINT,
+    SECONDARY_THUMBPRINT,
+    DATAPLANE_AUTH_TYPES,
+    DEVICE_TYPES,
+)
+
+settings = DynamoSettings(req_env_set=ENV_SET_TEST_IOTHUB_REQUIRED, opt_env_set=ENV_SET_TEST_IOTHUB_OPTIONAL)
+CWD = os.path.dirname(os.path.abspath(__file__))
+
+
+class TestHubExportImport(IoTLiveScenarioTest):
+    def __init__(self, test_case):
+        super(TestHubExportImport, self).__init__(test_case)
+        self.dest_hub = settings.env.azext_iot_desthub or "test-hub-" + generate_generic_id()
+        self.dest_hub_rg = settings.env.azext_iot_destrg or settings.env.azext_iot_testrg
+
+        # create destination hub
+
+        if not settings.env.azext_iot_desthub:
+            self.create_hub(self.dest_hub, self.entity_rg)
```

Comment:
Is this hub being torn down at the end of tests? Make sure you add this to teardown.

Also would help if you added a test tag (look how the __init__ adds a test tag at the end - it will just add a tag to tell us that this test was run x amount of times, very helpful if something breaks and resources are still there or if you are using your own hub and you want to keep track of what you do to it)

Reply:
Yes, if the hub is created here then it is torn down at the end.